views/layout.dt

```diff
@@ -16,7 +16,7 @@ html
 			.helper
 				.helper.expand-container.active
 					.logo
-						a(href=".")
+						a(href="..")
```

Comment:
`/` is shorter though, but `#{req.rootDir}` seems more standard. Which would be preferred? I assume the latter?